web/crux/src/domain/agent.ts

```diff
@@ -357,3 +361,13 @@ export type AgentToken = {
   iss: string
   iat: number
 }
+
+export type AgentEvent = {
+  id: string
+  status: NodeConnectionStatus
+  address?: string
+  version?: string
```

Comment:
There are use cases, where we don't have agent versions? cc. @nandor-magyar

Reply:
Not really,  but if the agent isn't connected we can't send its address, hence it's optional.